Marlin/src/lcd/extui/anycubic_vyper/dgus_tft.cpp

```diff
@@ -723,37 +723,31 @@ namespace Anycubic {
     TFTSer.println();
   }
 
-  void DgusTFT::SendValueToTFT(uint32_t value, uint32_t address) {
-    uint8_t *a_u8 = (uint8_t *)(&address),
-            *v_u8 = (uint8_t *)(&value);
-    uint8_t data[] = { 0x5A, 0xA5, 0x05, 0x82, a_u8[1], a_u8[0], v_u8[1], v_u8[0] };
+  void DgusTFT::SendValueToTFT(const uint16_t value, const uint32_t address) {
+    uint8_t data[] = { 0x5A, 0xA5, 0x05, 0x82, uint8_t(address >> 8), uint8_t(address & 0xFF), uint8_t(value >> 8), uint8_t(value & 0xFF) };
```

Comment:
@Bob-the-Kuhn — Compare the old behavior to the new behavior with some serial logging. There is a clear assumption here that only the low 16 bits of the `value` and the `address` are relevant, and since the old code was looking at the first two bytes the assumption is that the MCU is little-endian. Thus `a_u8[1]` must be the high order address byte and `a_u8[0]` is the low order byte. The same must be true for the value. But serial logging should more firmly establish that the result is the same or not.

So, I will start by scanning this commit for any obvious typos.